app/controllers/external_users/claims_controller.rb

```diff
@@ -155,6 +155,8 @@ def edit
   end
 
   def create
+    @claim = resource_klass.new(params_with_external_user_and_creator)
+    authorize! :create, @claim
```

Comment:
[Ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/2274/files#r174404793)